### PR TITLE
[C#] Extract Concepts from v2 exercise: rna-transcriptions

### PR DESCRIPTION
**Pull request by maurelio1234**
Still a work in progress, not ready for review.

Working on #623

### CODE REVIEW

**Comment by maurelio1234** [comment]
Hi, 

Just pushed a more complete draft, I hope it's enough to start a discussion around the PR.

I have a couple of questions:

1. what's the point of the table in "Concept interpretation" here?

https://github.com/maurelio1234/v3/tree/patch-2/languages/csharp/reference

why is it called "Concept interpretation"?

2. What do you think about adding things like "comments" and coding style/conventions as concepts?

File: languages/csharp/reference/exercise-concepts/rna-trascriptions.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# RNA transcription
+
```
Comment (ErikSchierboom) [inline languages/csharp/reference/exercise-concepts/rna-trascriptions.md]: Could you add a link to the implementation this is based on? See [this example](https://github.com/exercism/v3/blob/master/languages/csharp/reference/exercise-concepts/robot-name.md#concepts-of-robot-name).

**Comment by ErikSchierboom** [comment]
> what's the point of the table in "Concept interpretation" here?
https://github.com/maurelio1234/v3/tree/patch-2/languages/csharp/reference

The "Concept interpretation" table is there to describe in detail how the Concepts of Concept Exercises are mapped to C# Concepts. This list has a direct relation to the Concepts that are specified for the Concept Exercises in the `config.json` file: https://github.com/exercism/v3/blob/master/languages/csharp/config.json

> why is it called "Concept interpretation"?

Because Concepts are very broad and offer little detail. For the Concept Exercises though, it is important to keep track of how a particular Concept has been interpreted, as in what things does a Concept teach.

> What do you think about adding things like "comments" and coding style/conventions as concepts?

I feel that comments definitely are a separate Concept. The coding style/conventions stuff is a little bit more hard, but I don't mind including it for now. We can discuss this later.

File: languages/csharp/reference/exercise-concepts/rna-trascriptions.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Object-oriented
+
+- Classes: used on the template. Some people define a static dictionary to pre compute RNA pairs.
+- Encapsulation: used on the template. There are public elements, some people create private elements
```
Comment (robkeim) [inline languages/csharp/reference/exercise-concepts/rna-trascriptions.md]: For consistency with the rest of the file
```suggestion
- Encapsulation: used on the template. There are public elements, some people create private elements.
```

**Comment by maurelio1234** [comment]
> I feel that comments definitely are a separate Concept. The coding style/conventions stuff is a little bit more hard, but I don't mind including it for now. We can discuss this later.

Ok, it looks like the language side of the descriptions in `reference/concept` right?`

**Pull request merged**